common/djangoapps/student/views.py

```diff
@@ -402,7 +402,7 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
                     cert_status['download_url']
                 )
 
-    if status in ('generating', 'ready', 'notpassing', 'restricted', 'auditing', 'unverified'):
+    if status in set(['generating', 'ready', 'notpassing', 'restricted', 'auditing', 'unverified']):
```

Comment:
The set([]) notation is used to support python 2.6 and earlier, which didn't have the curly-brace set literal notation.  The old way is slower, as it builds a list, then calls the set function to convert the list to a set, and subject to variable shadowing issues (`set = lambda x: None; set([1, 2])`).  You can use the `dis` module to disassemble python bytecode, and see how they work:

```python
>>> def oldsetlit():
...     return set([1,2])
... 
>>> import dis
>>> dis.dis(oldsetlit)
  2           0 LOAD_GLOBAL              0 (set)
              3 LOAD_CONST               1 (1)
              6 LOAD_CONST               2 (2)
              9 BUILD_LIST               2
             12 CALL_FUNCTION            1
             15 RETURN_VALUE        
>>> def newsetlit():
...     return {1, 2}
... 
>>> dis.dis(newsetlit)
  2           0 LOAD_CONST               1 (1)
              3 LOAD_CONST               2 (2)
              6 BUILD_SET                2
              9 RETURN_VALUE        
```